### DISCUSSION TITLE
ltsp init masks real domain name via /etc/hosts - Why? How to avoid?

### DISCUSSION THREAD

**Discussion by shredderique** [ltsp/ltsp#784] [Q&A]
ltsp init masks the correct dns lookup by that last entry

```
# Generated by `ltsp init`, see man:ltsp(8)
127.0.0.1       localhost
127.0.1.1       pxe-h10-sko.zkrd.de zkrd-h10.de pxe-h10-sko
172.20.5.56     server
```
The masked IP corresponds to the `SERVER = 172.20.5.56` setting in /etc/ltsp/ltsp.conf which is supposed to resolve to the ltsp hosting server itself. How can I avoid this entry? Why is there?

thank you kindly!

**Comment by alkisg** [comment]
This entry needs to be there for SSH authentication to work.
You can create an additional one for the same IP with the HOSTS_x ltsp.conf directive:

https://ltsp.org/man/ltsp.conf/#HOSTS_x

**Comment by shredderique** [comment]
Thank you, this is a simple straightforward solution
